Dockerfile

```diff
@@ -1,19 +1,23 @@
 # ---- Base image - order important ----
-FROM hmctspublic.azurecr.io/ccd/definition-processor:dbyccu as base
+FROM hmctspublic.azurecr.io/ccd/definition-processor:latest as base
+USER root
+RUN usermod -u 1000 hmcts && \
```

Comment:
shouldn't we change this back to 1000 here then?
https://github.com/hmcts/ccd-definition-processor/blob/a217ea91595b403b90ad7fd88f8c9ff9764dd059/Dockerfile#L9